<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormEditUser</class>
 <widget class="QWidget" name="FormEditUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>612</width>
    <height>256</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="lbTipo">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>110</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Tipo:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leNombre">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="leCedula">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>10</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="lbCedula">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>10</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Ingrese Cedula:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnAceptar">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>200</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Editar</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbApellido">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>70</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Apellido:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbNombre">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>73</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Nombre:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="leApellido">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>70</y>
     <width>151</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnCancelar">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Buscar</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbProf">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>110</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Profesor</string>
   </property>
  </widget>
  <widget class="QRadioButton" name="rbEstu">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>140</y>
     <width>93</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Estudiante</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
